hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/bucket/FileIOEngine.java

```diff
@@ -267,6 +318,98 @@ void refreshFileConnection(int accessFileNum, IOException ioe) throws IOExceptio
     }
   }
 
+  @Override
+  public byte[] readPersistenceChecksum(String persistencePath) {
```

Comment:
this can be called readPersistentCheckSum(). But do we need this API. Instead can we just do this internally under verifyCheckSum() or verifyFileIntegrity()?

Reply:
It has been modified. Thanks for giving advice.